api/endpoints/cmr.py

```diff
@@ -24,6 +25,130 @@
 ns = api.namespace('cmr', description='Operations related to CMR')
 
 
+@ns.route('/granules')
```

Comment:
This CmrGranules class is a duplicate of the same class on line 330

Reply:
Reverted the cmr.py to whats on develop.